bin/runtime/Cargo.toml

```diff
@@ -30,7 +30,7 @@ pallet-identity = { workspace = true }
 pallet-multisig = { workspace = true }
 pallet-nomination-pools = { workspace = true }
 pallet-nomination-pools-runtime-api = { workspace = true }
-pallet-randomness-collective-flip = { workspace = true }
+pallet-insecure-randomness-collective-flip = { workspace = true }
```

Comment:
here: https://github.com/paritytech/substrate/tree/master/frame/insecure-randomness-collective-flip they are screaming that: **DO NOT USE IN PRODUCTION** - should we be worried? or it has always been like this

Reply:
it has always been like this. afaik it is only used in the pallet_contracts and source of randomness

Reply:
Is it still? I believe there was an effort to remove it after last audit.

Reply:
yep, this is deprecated so we dont have to worry about it being insecure. I can remove the randomness pallet from the runtime but lets do it in separate pr
![image](https://github.com/Cardinal-Cryptography/aleph-node/assets/27210860/b8f56818-0edc-4153-bcc3-3ff349271c56)

Reply:
sounds good for me, resolving